packages/common/mui/src/lib/area-chart/area-chart.tsx

```diff
@@ -0,0 +1,175 @@
+}
+
+export const AreaChart = ({
+  areaChartData,
+  CustomTooltip,
+  areaHeaderData,
+}: AreaChartProps) => {
+  const isLine = areaChartData[0]?.line;
+  const theme = useTheme();
+  const intl = useIntl();
+
+  const xAxisTickHandler = (v: number, i: number) => {
+    let result = '';
+    if (typeof v === 'number') {
+      const prevMonth =
+        i > 0
+          ? intl.formatDate(areaChartData[i - 1]['timestamp'] * 1000, {
+              month: 'short',
+              year: 'numeric',
+            })
+          : '';
+      const thisMonth = intl.formatDate(v * 1000, {
+        month: 'short',
+        year: 'numeric',
+      });
+
+      // Only label the axis on the first month
+      result = prevMonth !== thisMonth ? thisMonth : '';
+    }
+    return result;
+  };
+
+  const CustomTickHandler = (props) => {
+    const {
+      x,
+      y,
+      payload: { value },
+    } = props;
+
+    return (
+      <g transform={`translate(${x},${y})`} cursor={'pointer'}>
+        <text x={0} y={0} dy={16} textAnchor="center">
+          {intl.formatDate(value * 1000, {
+            month: 'short',
+            year: 'numeric',
+          })}
+        </text>
+      </g>
+    );
+  };
+
+  return (
+    <TradeSummaryBox sx={{ width: '100%' }}>
+      {areaChartData && areaChartData?.length > 0 ? (
+        <ChartContainer>
+          {areaHeaderData && <AreaChartHeader {...areaHeaderData} />}
+          <ResponsiveContainer width="100%" height={400}>
+            <ComposedChart
+              height={200}
+              data={areaChartData}
+              margin={{ top: 30, right: 30, left: 20, bottom: 20 }}
+            >
+              <Tooltip content={CustomTooltip} position={{ y: 0 }} />
+              <XAxis
+                dataKey="timestamp"
+                tickCount={0}
+                axisLine={false}
+                tickSize={0}
+                tickMargin={20}
+                domain={[
+                  (dataMin: number) => dataMin - ONE_WEEK,
+                  (dataMax: number) => dataMax + ONE_WEEK,
+                ]}
+                style={{ fill: theme.palette.typography.light }}
+                interval={0}
+                tickFormatter={isLine ? xAxisTickHandler : undefined}
+                tick={!isLine ? <CustomTickHandler /> : undefined}
+                tickLine={false}
+              />
+              <YAxis
+                orientation="left"
+                yAxisId={0}
+                padding={{ top: 8 }}
+                tickCount={6}
+                tickMargin={12}
+                width={60}
+                tickSize={0}
+                tickLine={false}
+                axisLine={false}
+                style={{ fill: theme.palette.typography.light }}
+                tickFormatter={(v: number) => `${v.toFixed(2)}%`}
+              />
+
+              <Line
+                type="monotone"
+                dataKey="line"
+                stroke={theme.palette.success.main}
```

Comment:
Similar here, I think stroke lines may be different for different charts (error vs success perhaps). Maybe align the prop such that the legend and the stroke here receive the same color value.

Reply:
I am working on making a reusable tool tip. Do we want the tool tip formatting to be the same across all chart. Example, all caps and all the same font weight? That's what the newest mocks show. Or do we want to keep the lower case and bold on strategy like the vault table has currently?